README.md

```diff
@@ -109,20 +109,26 @@ To compile gqrx from source you need the following dependencies:
     - Network
     - Widgets
     - Svg (runtime-only)
-- cmake version >= 3.2.0
+- cmake version >= 3.13.0
```

Comment:
The main CMakeLists file only requires cmake >= 3.5. Have you increased this due to an issue in the CMakeLists files, or because of the build commands you've suggested below?

I do see that `--parallel` was only added in cmake 3.12.

Reply:
In according with [link](https://cmake.org/cmake/help/latest/manual/cmake.1.html),  "-B <path-to-build>" added in version 3.13.
P.S. And CMake 3.13 is very old software. For example [Qt project](https://github.com/qt/qtbase/blob/a2e60ebee3737548d1be14fdbb39b08c515ae602/CMakeLists.txt#L8) for now use 3.16